tests/rptest/tests/partition_move_interruption_test.py

```diff
@@ -261,14 +329,14 @@ def new_controller():
         # self.redpanda.start_node(self.redpanda.get_node(to_stop))
 
         # wait for previous assignment to be set
-        def cancelled():
+        def cancelling():
```

Comment:
here actually "cancelled" looks more correct, because it waits until cancel process is finished